pkg/fdbstatus/status_checks.go

```diff
@@ -110,6 +110,7 @@ func getRemainingAndExcludedFromStatus(logger logr.Logger, status *fdbv1beta2.Fo
 	}
 
 	// Check in the status output which processes are already marked for exclusion in the cluster
+	// this should be cross-DC info bc the totals are all cluster-wide
```

Comment:
There is no need for doing cross-dc as the provided address will be local to one DC.

Reply:
It does still seem odd to me to isolate within a helper function which is called in a path from the admin cli client (though this code is only called outside of tests in removals, which does only work on the DC)

Reply:
discussed offline and agreed to not filter